indigo/release/Dockerfile

```diff
@@ -5,7 +5,6 @@ FROM osrf/ros:indigo-desktop
 MAINTAINER Dave Coleman dave@dav.ee
 
 # apt-commands are combined in single RUN statement with lists folder removal to reduce image size
-RUN apt-get update \
- && apt-get upgrade -y \
- && apt-get install -y ros-indigo-moveit-full \
- && rm -rf /var/lib/apt/lists/*
\ No newline at end of file
+RUN apt-get update && apt-get install -y \
+      ros-${ROS_DISTRO}-moveit-full \
```

Comment:
So I assume ${ROS_DISTRO} is set from the `` FROM osrf/ros:indigo-desktop`` image? cool